### PR TITLE
Changelog entry for v0.4.1

### PR DESCRIPTION
**Pull request by core-man**
**Description of proposed changes**

This PR is a draft changelog entry for v0.4.1.

**Preview**: https://pygmt-git-changelog-release-gmt.vercel.app/changes.html#changelog

Previous release at #1341

Changelog made by following the instructions at https://github.com/GenericMappingTools/pygmt/blob/main/doc/maintenance.md#updating-the-changelog

TODO:

- [x] Update the DOI badge
- [x] Update citation information https://github.com/GenericMappingTools/pygmt#citing-pygmt
- [x] Add the documentation link https://github.com/GenericMappingTools/pygmt#documentation-for-other-versions
- [x] Add highlights
- [x] Update with any remaining PRs merged before release
- [x] Edit the change list to remove any trivial changes (updates to the README, typo fixes, CI configuration, etc)
- [x] Edit the list of people who contributed to the release, linking to their GitHub account. Sort their names by the number of commits made since the last release (e.g., use git shortlog HEAD...v0.1.2 -sne).

Relates to #1402

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

File: doc/changes.md [hunk 1]
```diff
@@ -1,5 +1,54 @@
...
+* Add development toctree and PyGMT team page ([#1308](https://github.com/GenericMappingTools/pygmt/pull/1308))
+* Restructure contributing.md to separate docs/general info from contributing code section ([#1339](https://github.com/GenericMappingTools/pygmt/pull/1339))
+
+### Maintenance
+
+* Add tomli as a dependency in GMT Dev Tests ([#1401](https://github.com/GenericMappingTools/pygmt/pull/1401))
+* Change references from "master" branch to "main" branch ([#1360](https://github.com/GenericMappingTools/pygmt/pull/1360))
+* NEP29: Test PyGMT on NumPy 1.21 ([#1355](https://github.com/GenericMappingTools/pygmt/pull/1355))
+
+### Contributors
```
Comment (core-man) [inline doc/changes.md:37]:
```
$ git shortlog HEAD...v0.4.0 -sne
     5  Meghan Jones <meghanj@alum.mit.edu>
     3  Will Schlitzer <schlitzer90@gmail.com>
     2  Wei Ji <23487320+weiji14@users.noreply.github.com>
     1  Dongdong Tian <seisman.info@gmail.com>
     1  Kadatatlu Kishore <vskishore408@gmail.com>
     1  Michael Grund <23025878+michaelgrund@users.noreply.github.com>
     1  Sean H <31742569+sean0921@users.noreply.github.com>
     1  Soham Banerjee <63705023+soham4abc@users.noreply.github.com>
     1  Yao Jiayuan <coreman.seism@gmail.com>
     1  yohaimagen <mayochay@gmail.com>
```

File: README.rst [hunk 2]
```diff
@@ -154,7 +154,7 @@ Developers". Feel free to cite our work in your research using the following Bib
 
 .. code-block::
```
Comment (core-man) [inline README.rst:156]: Based on the number of commits to the repository, I did not change the order of authors.
```
$ git shortlog -sne
  280  Dongdong Tian <seisman.info@gmail.com>
   256  Leonardo Uieda <leouieda@gmail.com>
   143  Wei Ji <23487320+weiji14@users.noreply.github.com>
    86  Will Schlitzer <schlitzer90@gmail.com>
    42  Meghan Jones <meghanj@hawaii.edu> + <meghanj@alum.mit.edu>
    42  Michael Grund <23025878+michaelgrund@users.noreply.github.com> + <michael_grund@gmx.de>
    31  Yao Jiayuan <coreman.seism@gmail.com>
    15  Liam Toney <38269494+liamtoney@users.noreply.github.com>
     3  kmaterna <kathrynmaterna@gmail.com>
     2  Abhishek Anant <81291713+itsabhianant@users.noreply.github.com>
     2  Tyler Newton <54011699+tjnewton@users.noreply.github.com>
    1  Malte Ziebarth <malte.ziebarth@fmvkb.de>
```

File: README.rst [hunk 3]
```diff
@@ -169,12 +169,12 @@ Developers". Feel free to cite our work in your research using the following Bib
                       Ziebarth, Malte and
                       Wessel, Paul},
       title        = {{PyGMT: A Python interface for the Generic Mapping Tools}},
-      month        = jun,
+      month        = jul,
```
Comment (weiji14) [inline README.rst:172]: It's August already :sweat_smile:
```suggestion
      month        = aug,
```

File: doc/changes.md [hunk 4]
```diff
@@ -1,5 +1,54 @@
...
+* A gallery example for the contour method ([#1387](https://github.com/GenericMappingTools/pygmt/pull/1387))
+* Add common option aliases to COMMON_OPTIONS in decorators.py ([#1407](https://github.com/GenericMappingTools/pygmt/pull/1407))
+* Add return statement to grdclip and grdgradient docstring ([#1390](https://github.com/GenericMappingTools/pygmt/pull/1390))
+* Add development toctree and PyGMT team page ([#1308](https://github.com/GenericMappingTools/pygmt/pull/1308))
+* Restructure contributing.md to separate docs/general info from contributing code section ([#1339](https://github.com/GenericMappingTools/pygmt/pull/1339))
+
+### Maintenance
+
+* Add tomli as a dependency in GMT Dev Tests ([#1401](https://github.com/GenericMappingTools/pygmt/pull/1401))
+* Change references from "master" branch to "main" branch ([#1360](https://github.com/GenericMappingTools/pygmt/pull/1360))
```
Comment (weiji14) [inline doc/changes.md:34]: This "master" to "main" rename should be a highlight item.

File: doc/changes.md [hunk 5]
```diff
@@ -1,5 +1,54 @@
...
+
+### Documentation
+
+* Add gallery example for grdclip ([#1396](https://github.com/GenericMappingTools/pygmt/pull/1396))
+* Add a gallery example for different colormaps in subplots ([#1394](https://github.com/GenericMappingTools/pygmt/pull/1394))
+* Add a gallery example showing individual custom symbols ([#1348](https://github.com/GenericMappingTools/pygmt/pull/1348))
+* A gallery example for the contour method ([#1387](https://github.com/GenericMappingTools/pygmt/pull/1387))
+* Add common option aliases to COMMON_OPTIONS in decorators.py ([#1407](https://github.com/GenericMappingTools/pygmt/pull/1407))
+* Add return statement to grdclip and grdgradient docstring ([#1390](https://github.com/GenericMappingTools/pygmt/pull/1390))
+* Add development toctree and PyGMT team page ([#1308](https://github.com/GenericMappingTools/pygmt/pull/1308))
```
Comment (weiji14) [inline doc/changes.md:28]: Maybe have this as a highlight item? Would be nice to link to https://www.pygmt.org/dev/team.html too.

File: doc/changes.md (hunk 1 above)
Comment (weiji14) [reply doc/changes.md:37]: @core-man, you'll have 1 more commit after merging in this changelog PR, so technically you can move up the list above @seisman :stuck_out_tongue_closed_eyes:

File: doc/changes.md [hunk 6]
```diff
@@ -1,5 +1,54 @@
...
+* [Meghan Jones](https://github.com/meghanrjones)
+* [Will Schlitzer](https://github.com/willschlitzer)
+* [Wei Ji Leong](https://github.com/weiji14)
+* [Dongdong Tian](https://github.com/seisman)
+* [Kadatatlu Kishore](https://github.com/kadatatlukishore)
+* [Michael Grund](https://github.com/michaelgrund)
+* [@sean0921](https://github.com/sean0921)
+* [Soham Banerjee](https://github.com/soham4abc)
+* [Jiayuan Yao](https://github.com/core-man)
+* [@yohaimagen](https://github.com/yohaimagen)
```
Comment (weiji14) [inline doc/changes.md:48]: Ping @sean0921 and @yohaimagen to see if they would like their full name in the list. If not we can just go by their GitHub username.

**Review by weiji14 — COMMENTED** [review]
Looking good! Just some comments, mostly on the highlight items.

File: doc/changes.md (hunk 5 above)
Comment (core-man) [reply doc/changes.md]: Done in 32733a6.

File: doc/changes.md (hunk 4 above)
Comment (core-man) [reply doc/changes.md]: Done in b903279

File: doc/changes.md [hunk 7]
```diff
@@ -1,5 +1,54 @@
 # Changelog
 
+## Release v0.4.1 (2021/08/07)
+
+[![Digital Object Identifier for PyGMT v0.4.1](https://zenodo.org/badge/DOI/10.5281/zenodo.5162003.svg)](https://doi.org/10.5281/zenodo.5162003)
+
+### Highlights
+
+* 🎉 **Patch release with multiple gallery examples** 🎉
+* Change references from "master" branch to "main" branch ([#1360](https://github.com/GenericMappingTools/pygmt/pull/1360))
```
Comment (seisman) [inline doc/changes.md:10]: > Change references from "master" branch to "main" branch

Change the above sentence to:

> Change the default branch name from "master" to "main" to increase inclusivity

File: doc/changes.md [hunk 8]
```diff
@@ -1,5 +1,54 @@
...
+* Add development toctree and [PyGMT team page](https://www.pygmt.org/latest/team.html) ([#1308](https://github.com/GenericMappingTools/pygmt/pull/1308))
```
Comment (seisman) [inline doc/changes.md:11]: I feel people may not understand what "development toctree" means. Perhaps we can just say:

> Add a "PyGMT Team" page

File: doc/changes.md [hunk 9]
```diff
@@ -1,5 +1,54 @@
...
+* Add common alias "verbose" (V) to grdlandmask and savefig ([#1343](https://github.com/GenericMappingTools/pygmt/pull/1343))
+
+### Bug Fixes
+
+* Change invalid input conditions in grdtrack ([#1376](https://github.com/GenericMappingTools/pygmt/pull/1376))
+* Fix bug so that x2sys_cross accepts dataframes with NaN values ([#1369](https://github.com/GenericMappingTools/pygmt/pull/1369))
+
+### Documentation
+
+* Add gallery example for grdclip ([#1396](https://github.com/GenericMappingTools/pygmt/pull/1396))
```
Comment (seisman) [inline doc/changes.md:24]:
```suggestion
* Add a gallery example for grdclip ([#1396](https://github.com/GenericMappingTools/pygmt/pull/1396))
```

File: doc/changes.md [hunk 10]
```diff
@@ -1,5 +1,54 @@
...
+* Add a gallery example for different colormaps in subplots ([#1394](https://github.com/GenericMappingTools/pygmt/pull/1394))
+* Add a gallery example showing individual custom symbols ([#1348](https://github.com/GenericMappingTools/pygmt/pull/1348))
+* A gallery example for the contour method ([#1387](https://github.com/GenericMappingTools/pygmt/pull/1387))
```
Comment (seisman) [inline doc/changes.md:27]:
```suggestion
* Add a gallery example for the contour method ([#1387](https://github.com/GenericMappingTools/pygmt/pull/1387))
```

File: doc/changes.md (hunk 8 above)
Comment (core-man) [reply doc/changes.md]: Done in https://github.com/GenericMappingTools/pygmt/pull/1414/commits/5426bbef085c3de95b2401e457992230ecfe5aee

File: doc/changes.md (hunk 7 above)
Comment (core-man) [reply doc/changes.md]: Done in https://github.com/GenericMappingTools/pygmt/pull/1414/commits/5426bbef085c3de95b2401e457992230ecfe5aee

File: README.rst [hunk 11]
```diff
@@ -221,6 +221,7 @@ Documentation for other versions
 * `Development <https://www.pygmt.org/dev>`__ (reflects the *master* branch on
```
Comment (michaelgrund) [inline README.rst:221]:
```suggestion
* `Development <https://www.pygmt.org/dev>`__ (reflects the *main* branch on
```
Should be also adjusted here, right?

File: doc/changes.md (hunk 6 above)
Comment (yohaimagen) [reply doc/changes.md]: I would like to use my full name please, Yohai Magen, thanks!

Comment (core-man) [reply doc/changes.md]: > I would like to use my full name please, Yohai Magen, thanks!

Thanks. Done in 5578ece.

**Review by seisman — APPROVED** [review]
Looks good to me after merging #1415.

**Review by michaelgrund — APPROVED** [review]
Ready to merge after #14

File: README.rst [hunk 12]
```diff
@@ -169,12 +169,12 @@ Developers". Feel free to cite our work in your research using the following Bib
                       Ziebarth, Malte and
```
Comment (core-man) [inline README.rst:169]: We might want to add **Yohai Magen** here if #1417 is merged before the release.

File: doc/changes.md [hunk 13]
```diff
@@ -1,5 +1,54 @@
...
+* Add common alias "verbose" (V) to grdlandmask and savefig ([#1343](https://github.com/GenericMappingTools/pygmt/pull/1343))
+
+### Bug Fixes
+
+* Change invalid input conditions in grdtrack ([#1376](https://github.com/GenericMappingTools/pygmt/pull/1376))
+* Fix bug so that x2sys_cross accepts dataframes with NaN values ([#1369](https://github.com/GenericMappingTools/pygmt/pull/1369))
+
+### Documentation
+
+* Add a gallery example for grdclip ([#1396](https://github.com/GenericMappingTools/pygmt/pull/1396))
```
Comment (weiji14) [inline doc/changes.md:24]:
```suggestion
* Combine documentation and compatibility sections in README.rst (([#1415](https://github.com/GenericMappingTools/pygmt/pull/1415))
* Add a gallery example for grdclip ([#1396](https://github.com/GenericMappingTools/pygmt/pull/1396))
```

File: doc/changes.md [hunk 14]
```diff
@@ -1,5 +1,54 @@
 # Changelog
 
+## Release v0.4.1 (2021/08/07)
+
+[![Digital Object Identifier for PyGMT v0.4.1](https://zenodo.org/badge/DOI/10.5281/zenodo.5162003.svg)](https://doi.org/10.5281/zenodo.5162003)
+
+### Highlights
+
+* 🎉 **Patch release with multiple gallery examples** 🎉
+* Change the default branch name from "master" to "main" to increase inclusivity ([#1360](https://github.com/GenericMappingTools/pygmt/pull/1360))
```
Comment (weiji14) [inline doc/changes.md:10]:
```suggestion
* Change default GitHub branch name from "master" to "main" to increase inclusivity ([#1360](https://github.com/GenericMappingTools/pygmt/pull/1360))
```

File: README.rst (hunk 12 above)
Comment (core-man) [reply README.rst:169]: done in https://github.com/GenericMappingTools/pygmt/pull/1414/commits/acbf3398835d5378f045bf13d5f3e5fd188db425.

File: doc/changes.md (hunk 1 above)
Comment (weiji14) [reply doc/changes.md:37]: Need to reorder this list again. Running `git shortlog HEAD...v0.4.0 -sne` on the main branch at 5bcae59cfc492853036c54d29a4216d184c0dfa5 shows the below. You'll need to move @michaelgrund and @yohaimagen up a few spots:

```
     8  Meghan Jones <meghanj@alum.mit.edu>
     6  dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
     4  Will Schlitzer <schlitzer90@gmail.com>
     3  Michael Grund <23025878+michaelgrund@users.noreply.github.com>
     3  Wei Ji <23487320+weiji14@users.noreply.github.com>
     2  yohaimagen <mayochay@gmail.com>
     1  Dongdong Tian <seisman.info@gmail.com>
     1  Kadatatlu Kishore <vskishore408@gmail.com>
     1  Sean H <31742569+sean0921@users.noreply.github.com>
     1  Soham Banerjee <63705023+soham4abc@users.noreply.github.com>
     1  Yao Jiayuan <coreman.seism@gmail.com>
```

Comment (core-man) [reply doc/changes.md:38]: Great~ done in https://github.com/GenericMappingTools/pygmt/pull/1414/commits/3e364cc8777bdb90cbfa7e6aa75a41bd0bcbf2df.

File: doc/changes.md [hunk 15]
```diff
@@ -1,5 +1,55 @@
...
+* Add common alias "verbose" (V) to grdlandmask and savefig ([#1343](https://github.com/GenericMappingTools/pygmt/pull/1343))
+
+### Bug Fixes
+
+* Change invalid input conditions in grdtrack ([#1376](https://github.com/GenericMappingTools/pygmt/pull/1376))
+* Fix bug so that x2sys_cross accepts dataframes with NaN values ([#1369](https://github.com/GenericMappingTools/pygmt/pull/1369))
+
+### Documentation
+
+* Combine documentation and compatibility sections in README.rst (([#1415](https://github.com/GenericMappingTools/pygmt/pull/1415))
```
Comment (weiji14) [inline doc/changes.md:24]: Oops, extra bracket here, my bad!

```suggestion
* Combine documentation and compatibility sections in README.rst ([#1415](https://github.com/GenericMappingTools/pygmt/pull/1415))
```

**Review by weiji14 — COMMENTED** [review]
Ok, looks good to me aside from one minor typo... Do we want to wait a bit longer, or merge this in and get on with tagging the release? ~~If no objections I'll merge this in <1 hour at UTC 11:00?~~

**Comment by core-man** [comment]
> Ok, looks good to me aside from one minor typo... Do we want to wait a bit longer, or merge this in and get on with tagging the release?

Maybe we could still wait for about a few hours to see if **Will** is okay?

**Comment by weiji14** [comment]
> > Ok, looks good to me aside from one minor typo... Do we want to wait a bit longer, or merge this in and get on with tagging the release?
> 
> Maybe we could still wait for about a few hours to see if **Will** is okay?

Sounds good. Just want to make the 7 Aug 2021 deadline (specifically before it turns 8 Aug at UTC).

**Review by willschlitzer — APPROVED** [review]
Looks good to me!

**Comment by core-man** [comment]
> > > Ok, looks good to me aside from one minor typo... Do we want to wait a bit longer, or merge this in and get on with tagging the release?
> > 
> > 
> > Maybe we could still wait for about a few hours to see if **Will** is okay?
> 
> Sounds good. Just want to make the 7 Aug 2021 deadline (specifically before it turns 8 Aug at UTC).

Let's do it now~~

File: doc/changes.md (hunk 6 above)
Comment (sean0921) [reply doc/changes.md]: Sorry for being late.  Just change my full name to *Sean Li-Shin Ho* for documents in `main` branch, or I can make a PR for this if needed.  Thanks!

Comment (weiji14) [reply doc/changes.md]: Ok, I've edited in your full name at https://github.com/GenericMappingTools/pygmt/releases/tag/v0.4.1. Feel free to open a PR to change things in https://github.com/GenericMappingTools/pygmt/blob/v0.4.1/doc/changes.md#contributors (you're also welcome to add your name to [AUTHORS.md](https://github.com/GenericMappingTools/pygmt/blob/main/AUTHORS.md) if you're keen on contributing more in the future.

**Pull request merged**